pages/docs/overview.mdx

```diff
@@ -1,48 +1,51 @@
 import Section from "../../components/Section/Section";
 import {CodeTab, CodeTabs} from "../../components/CodeTabs/CodeTabs";
+import { Callout } from 'nextra-theme-docs';
 import Image from 'next/image'
 
 # Drizzle ORM  
-Drizzle ORM is a headless TypeScript ORM with a head.  
-It looks and feels simple, performs on day 1000 of your project, 
-lets you do things your way and helps when you need help.  
+> Drizzle is a good friend who's there for you when necessary and doesn't bother when you need some space.
 
-> Drizzle is a good friend who's there for you when necessary and doesn't bother when you need some space
+Drizzle ORM is a headless TypeScript ORM with a head 🐲
 
-It is the only ORM with both [relational](/docs/rqb) and [SQL-like](/docs/select) query APIs 
+It looks and feels simple, performs on day _1000_ of your project, 
+lets you do things your way, and is there when you need it.  
+
+**It's the only ORM with both [relational](/docs/rqb) and [SQL-like](/docs/select) query APIs**, 
 providing you best of both worlds when it comes to accessing your relational data. 
-Drizzle is lightweight, performant, typesafe, non lactose, gluten-free, flexible and serverless-ready **by design**.  
+Drizzle is lightweight, performant, typesafe, non lactose, gluten-free, sober, flexible and **serverless-ready by design**.
+Drizzle is not just a library, it's an experience 🤩
 
-  
 [![Drizzle bestofjs](/images/bestofjs.jpg)](https://bestofjs.org/projects/drizzle-orm)
 
 ## Headless ORM? 
-First and foremost Drizzle is a library and a set of complementary opt-in tools. 
-ORM stands for object relational mapping and developers tend to call Django-like or Spring-like tools an ORM. 
-We trully believe it's a misconception based on legacy and we call them Data Frameworks.
+First and foremost, Drizzle is a library and a collection of complementary opt-in tools. 
+
+**ORM** stands for _object relational mapping_, and developers tend to call Django-like or Spring-like tools an ORM. 
+We trully believe it's a misconception based on legacy nomenclature and we call them **data frameworks**.
 
-With data frameworks you have to build projects **around them** and not **with them**.  
-Drizzle lets you build your project the way you want or need, lets you be productive and does not interfere. 
-It lets you solve SQL problems with SQL toolkit and provide you a variety of opt in tools 
-to take your developer experience through the roof.  
+<Callout type="error" emoji="️💔">
```

Comment:
![image](https://github.com/drizzle-team/drizzle-orm-docs/assets/22394081/9b9ba566-c5cd-470e-947c-4902d7dcb6ed)

Reply:
Could you share more info? 

Not seeing this behavior on any browser I tested on or online screenshoting services.

https://pikwy.com/web/6504c434b5cdfb554b2758f2

![screenshot](https://api.pikwy.com/web/6504c434b5cdfb554b2758f2.jpg)

Reply:
@FTCHD Apologies for not adding more information. I'm on Firefox 117.0, MacBook Pro 13inch.
<img width="1680" alt="image" src="https://github.com/drizzle-team/drizzle-orm-docs/assets/22394081/8b3f36e6-a48b-4b2a-84a9-72b05d2442ab">